changes/ticket32604

```diff
@@ -0,0 +1,7 @@
+  o Minor features (onion services):
+    - Version 3 onion services can now use HiddenServiceExportRendPoint
+      to get first 12 chars of the rendezvous point's fingerprint.
+    - Version 3 onion services can now use HiddenServiceExportInstanceID
+      which enables them to differenciate traffic when multiple tor
+      instances share the same backend.
+    - Closes ticket 32604. Patch by Moonsik Park.
```

Comment:
This isn't quite how we format changes files. I'll suggest some changes on individual lines, but you'll need to re-wrap the lines.

Reply:
Each changelog entry can be sorted independently, so there needs to be a "Closes" line in each entry, not just the last entry. Closes lines do not start with "- ".
```suggestion
    - Closes ticket 32604. Patch by Moonsik Park.
```

Reply:
Fixed in `2582e28`.

Reply:
I think it's probably easiest if I fix the changes file?

Reply:
Fixed in 3a8dee4.